src/ucp/proto/proto_perf.c

```diff
@@ -430,6 +430,28 @@ ucs_status_t ucp_proto_perf_aggregate2(const char *name,
     return ucp_proto_perf_aggregate(name, perf_elems, 2, perf_p);
 }
 
+void ucp_proto_perf_apply_bias(ucp_proto_perf_t *perf, double bias)
```

Comment:
IMO it would be better to make it more generic, i.e something like
`void ucp_proto_perf_apply_func(ucp_proto_perf_t *perf, ucs_linear_func_t func, const char *name)`
`ucp_proto_perf_apply_func(perf, ucs_linear_func_make(0.0, 1.0 - bias), "bias");`